src/webportal/src/app/job/job-view/job-view.component.js

```diff
@@ -490,6 +493,18 @@ $(document).ready(() => {
     loadJobs(query['vcName']);
     $('#content-wrapper').css({'overflow': 'hidden'});
   }
+  $(document).on('click', '#resubmitjob_btn', () => {
+    let url;
+    let config_yaml = yaml.safeLoad(configInfo);
+    let username = $('#nav-username').text();
```

Comment:
Is this current login user but not current job's user?

Reply:
Yes, current login user only has the autority to resubmit his own job

Reply:
Where is the limitation of "own job", or what happens if user resubmit others job?